internal/service/searchindex/resource_search_index_test.go

```diff
@@ -11,7 +11,8 @@ import (
 	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/acc"
 )
 
-func TestAccSearchIndex_basic(t *testing.T) {
+func basicTestCase(t *testing.T) *resource.TestCase {
```

Comment:
knit: i think you normally pass testing.T not pointer

Reply:
Thanks. Updated ✅  in 78a605e